react-multiplatform-app/src/pages/Animation.tsx

```diff
@@ -0,0 +1,19 @@
+import {StyleSheet, View} from "react-native";
+
+const Animation = () => {
+
+
+  return <View style={styles.container}>
+
+  </View>
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+export default Animation
```

Comment:
File not ending with empty line. Is it ok in react project?

Reply:
Not necessary (eslint disabled)